virt/virtual_machines/virt-vtpm-devices.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: ASSEMBLY
+[id="virt-vtpm-devices_{context}"]
+= Virtual trusted platform module devices
+include::_attributes/common-attributes.adoc[]
+:context: virt-vtpm-devices
+
+toc::[]
+
+:FeatureName: Virtual trusted platform module devices
+
+A Virtual Trusted Platform Module (vTPM) device protects virtual machines
+(VMs) by storing secrets similarly to a physical Trusted Platform Module (TPM)
+hardware chip.
+
+[NOTE]
+====
+vTPM implementation is non-persistent. vTPM discards stored secrets
+after each shutdown.
+====
+
+You can use a vTPM device with VMs running any operating system. However, if you
+do not enable vTPM, then the VM does not recognize a TPM device, even if the
+node has one.
+
+[NOTE]
+====
+Windows 11 normally requires the presence of a physical TPM chip to install or
+boot. A vTPM device allows VMs created from a Windows 11 image to install or
+boot without having a physical TPM chip.
+====
```

Comment:
I think this might be better as body text (possibly in a separate concept module about vTPMs), because it's a great example of _why_ you would want to do this.